Bottom: b455846ab54b43f15e7f1f1e3e2263f1b7a48ce6
Top:    1cfb98adc0d8eebf1d0e401ff519028698412e28
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-15 18:01:15 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
